packages/plugin-ext/src/plugin/workspace.ts

```diff
@@ -395,4 +402,12 @@ export class WorkspaceExtImpl implements WorkspaceExt {
         return true;
     }
 
+    private async updateWorkspaceFile(): Promise<void> {
+        const workspace = await this.proxy.$getWorkspaceFile();
+
+        // A workspace directory imply an undefined workspace file
```

Comment:
minor:

```suggestion
        // A workspace directory implies an undefined workspace file.
```

Reply:
To be implemented in next commit, thanks !